java/java-common/src/main/java/org/cloudburstmc/protocol/java/data/text/ChatPosition.java

```diff
@@ -0,0 +1,13 @@
+package org.cloudburstmc.protocol.java.data.text;
+
+public enum ChatPosition {
+    CHAT_BOX,
+    SYSTEM,
+    GAME_INFO;
+
+    private static final ChatPosition[] VALUES = values();
```

Comment:
What is advantage of putting it to new static field over just using values()?

Reply:
`values()`, apparently, allocates a new array each time it is called.

Reply:
Although for an enum this small i doubt it would make a difference

Reply:
It's to prevent defensive copies as `values()` allocates a new array every instance. Doubt it'll make a meaningful difference here, but I've just done it for every enum so far so I did so here for the sake of consistency.